app/taskqueue/tasks/kcidb.py

```diff
@@ -40,10 +43,15 @@ def push_build(args):
 @taskc.app.task(name="kcidb-tests")
 def push_tests(group_id):
     kcidb_options = taskc.app.conf.get("kcidb_options")
+    pid = os.getpid()
+    submit_process = taskc.app.kcidb_pool.get(pid)
+    print(submit_process)
```

Comment:
Please drop `print` statements once this has been tested on staging, or convert to actual log messages.